.ci/containers/github-differ/test_tools.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+MM_LOCAL_PATH=$1
+TPG_LOCAL_PATH=$2
+mm_commit_sha=$3
+build_id=$4
+build_step=$5
+project_id=$6
+
+github_username=modular-magician
+
+pushd $MM_LOCAL_PATH/tools/breaking-change-detector
+go test
+exit_code=$?
+popd
```

Comment:
```suggestion
set +e
pushd $MM_LOCAL_PATH/tools/breaking-change-detector
go test
exit_code=$?
popd
set -e
```

We'll need to set the error code so that compilation issues and test failures don't stop execution of the script